spec/requests/v6/assessments_controller_spec.rb

```diff
@@ -35,6 +35,16 @@ module V6
           },
         ]
       end
+      let(:employment_income_without_payments_params) do
+        [
+          {
+            name: "Job 1",
+            client_id: SecureRandom.uuid,
+            receiving_only_statutory_sick_or_maternity_pay: true,
```

Comment:
Putting this flag into the test is probably a mistake. It's presence means that this test triggers the old behaviour, whereas we want new behaviour that doesn't give the £45 allowance when there are no payments